pymc3/backends/base.py

```diff
@@ -105,6 +138,47 @@ def get_values(self, varname, burn=0, thin=1):
         """
         raise NotImplementedError
 
+    def get_stats(self, varname, sampler_idx=None, burn=0, thin=1):
```

Comment:
I think I'd call this `get_sampler_stats()` as I'd assume `get_stats()` would get me stats of the posterior samples.